src/profile-logic/import/chrome.js

```diff
@@ -253,6 +258,14 @@ function getThreadInfo(
   );
   if (threadNameEvent) {
     thread.name = threadNameEvent.args.name;
+    if (thread.name.startsWith('Cr') && thread.name.endsWith('Main')) {
+      // Hack: Rename this thread to "GeckoMain" so that it gets detected as the
+      // main thread for the globalTrack of its process, and so that the UI
+      // displays a marker timeline.
+      // TODO: Replace the name detection with an isMainThread field on the thread.
```

Comment:
We should file this.

Reply:
filed issue #2508 and mentioned it in a comment